projects/vdk-plugins/vdk-meta-jobs/README.md

```diff
@@ -38,7 +39,8 @@ def run(job_input):
 When defining a job to be run following attributes are supported:
 * **job_name**: required, the name of the data job
 * **team_name:**: optional, the team of the data job. If omitted , it will use the meta job's team
-* **fail_meta_job_on_error**: optional, default is true. if true, the meta job will abort and fail if the orchestrated job fails, if false, meta job won't fail and continue.
+* **fail_meta_job_on_error**: optional, default is true. If true, the meta job will abort and fail if the orchestrated job fails, if false, meta job won't fail and continue.
+* **arguments**: optional, the arguments that are passed to the data job. The arguments could be used for parameter substitution in queries.
```

Comment:
`The arguments could be used for parameter substitution in queries.` 

That seems like it might be confusing here. In what queries in the DAG? In context of a dag. Just passing argument to underlying orchestrated data job seem enough

Reply:
Now that I look at it, it might be confusing, yes, changing it.